0x48piraj/index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+<html>
+    <head>
+        <title>Creative Flask Web Form</title>
+       <link rel="stylesheet" media="screen" href ="static/bootstrap.min.css">
+       <link rel="stylesheet" href="static/bootstrap-theme.min.css">
+       <meta name="viewport" content = "width=device-width, initial-scale=1.0">
+ 
+    </head>
+    <body>
+ 
+ 
+<div class="container">
+ 
+ 
+  <h2>Creative Flask Web Form</h2>
+  <form  action="" method="post" role="form">
+    {{ form.csrf }}
+    <div class="form-group">
+      <label for="name">Name:</label>
+      <input type="text" class="form-control" id="name" name="name" placeholder="FOSSASIA's Family Member's Name?">
+      <br>
+      <label for="twitter">Twitter: (optional)</label>
+      <input type="text" class="form-control" id="twitter" name="twitter" placeholder="What's your Twitter handle?">
+      <br>
+      <label for="git">Github: (optional)</label>
+      <input type="git" class="form-control" id="git" name="git" placeholder="What's your Github handle?">
+ 
+ 
+    </div>
+    <button type="submit" class="btn btn-success">Go!</button>
+  </form>
+ 
+  <br>
+        {% with messages = get_flashed_messages(with_categories=true) %}
+            {% if messages %}
+ 
+        {% for message in messages %}
+            {% if "Error" not in message[1]: %}
+                <div class="alert alert-info">
+                <strong>Yay! </strong> {{ message[1] }}
+                </div>
+            {% endif %}
+ 
+            {% if "Error" in message[1]: %}
+                <div class="alert alert-warning">
+                {{ message[1] }}
+                </div>
+            {% endif %}
+        {% endfor %}
+            {% endif %}
+        {% endwith %}
+ 
+</div>
+<br>            
```

Comment:
So instead of `<br>` you can define one class which specifies white space.

Reply:
Yes! But I used this to make the code look easier :wink: